internal/cli/data/action-template-credentials-exchange.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I'm not familiar with the /data/ files usage. Do you think we could have /cli/data/README.md explaining briefly?

Reply:
Sure, added in https://github.com/auth0/auth0-cli/pull/316/commits/4023c615561e84a418e494f6c4b9b2d4737e6a6b